firestore/proguard-rules.pro

```diff
@@ -0,0 +1 @@
+-dontwarn com.firebase.ui.firestore.paging.**
```

Comment:
Do we have some dependency from regular --> paging that is causing this?  I am surprised that we need to do this (but not opposed)

Reply:
Yeah, if someone doesn't use the paging library, proguard is going to complain about a bunch of missing classes.

Reply:
Ohh right.  Yeah that makes sense.